pom.xml

```diff
@@ -70,5 +70,23 @@
             <artifactId>workflow-step-api</artifactId>
             <version>1.15</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>1.15</version>
```

Comment:
:ant: use 2.x for all new dependencies (I am anyway migrating to 2.x for _existing_ dependencies)